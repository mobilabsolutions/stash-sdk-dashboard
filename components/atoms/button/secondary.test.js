/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import { withTestSetup } from '../../../test_utils'
import SecondaryButton from './secondary'

const TestButton = withTestSetup(SecondaryButton)

it('SecondaryButton Should Render', () => {
  const component = renderer.create(<TestButton label="Login" />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

it('Disabled SecondaryButton Should Render', () => {
  const component = renderer.create(<TestButton label="Disabled" disabled />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
