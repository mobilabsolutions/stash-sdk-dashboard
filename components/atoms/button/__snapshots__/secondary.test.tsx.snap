// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disabled SecondaryButton Should Render 1`] = `
<button
  className="secondary__Button-j7mjyk-0 iLEulF"
  disabled={true}
  onClick={null}
  type="button"
>
  <span>
    Disabled
  </span>
</button>
`;

exports[`SecondaryButton Should Render 1`] = `
<button
  className="secondary__Button-j7mjyk-0 gxdnVF"
  disabled={false}
  onClick={null}
  type="button"
>
  <span>
    Login
  </span>
</button>
`;
